scholly/base/src/components/publications/Publications.js

```diff
@@ -1,69 +1,70 @@
 /**
  * The Publications component displays a list of publications.
  */
-import React, { Fragment, useState } from 'react';
+import React, { Fragment } from 'react';
```

Comment:
@vgoh1 we should keep an eye out if the `Fragment` import throws a check when changes from main are merged with the updated SA. I expect it should.